system-parachains/collectives/collectives-polkadot/constants/src/lib.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot.  If not, see <http://www.gnu.org/licenses/>.
+
+#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
can we drop the `std` feature altogether for this crate?

Reply:
Do you mean also to remove from Cargo.toml:
```
[features]
default = ["std"]
std = []
```
and also usage:
```
std = [
    ...
    "collectives-polkadot-runtime-constants/std",
    ...
```
?

Actually, I don't know, I don't see any/much libraries used in the runtimes without this stuff. I don't know, can we remove it?

Reply:
afaict this is only used by runtime code which is compiled to wasm always without `std`, so it should work

Reply:
removed, yes, looks like everything is ok (at least locally), lets wait for the CI

Reply:
hmm, strange, CI complains: https://github.com/polkadot-fellows/runtimes/actions/runs/8111248007/job/22170156363?pr=182

Reply:
after `cargo clean, it does not work locally also, I had to revert it

Reply:
ok, at least we tried